### ISSUE TITLE
Errors encountered in careless_zero.py

### ISSUE THREAD

**Issue by DorisMai** [Hekstra-Lab/careless-examples#8]
I encountered a few errors when running the careles_zero example. I made the following 2 changes in careless_zero.py to get the script running -- perhaps this is some dependency version issue?

1) line 25: mtz.loc[:,['Hasu','Kasu','Lasu']] = mtz.hkl_to_asu().loc[:,['H', 'K', 'L']]
Changed to: mtz[['Hasu','Kasu','Lasu']] = mtz.hkl_to_asu().loc[:,['H', 'K', 'L']]
Comments: otherwise, all three columns are nan values, and all entries in mtz[‘miller_id’] are -1.

2) line 81: optimizer.minimize(minus_elbo, [q.trainable_variables , NN.trainable_variables])
Changed to:
	all_vars = [var for var in q.trainable_variables] + [var for var in NN.trainable_variables]
        optimizer.minimize(minus_elbo, all_vars) 
Comments: If I don't flatten the list, otherwise I get AttributeError: 'tuple' object has no attribute '_unique_id'

**Comment by kmdalton** [comment]
Thanks very much for this. Sorry that this went unnoticed. We don't have unit tests for `careless-examples`. Just for the `careless` CLI. I was able to confirm both your errors. I'll make a PR shortly. 

Specifics:
 1) Pandas has changed the syntax allowable in `.loc` since I wrote this example. 
 2) I can replicate your error on tf 2.11 but not tf 2.10. Support for nested structures must have disappeared from the `keras` Adam implementation between then.
